pages/installation/operating_system_packages.rst

```diff
@@ -23,11 +23,13 @@ Prerequisites
 
 Make sure to install and configure the following software before installing and starting any Graylog services:
 
-* Java (>= 8)
+* Java ( 8 or 11 )
```

Comment:
I think it was better before, you can also run the server with 9 or 10.

Reply:
ok, wasn't aware of that. My information: Java 8 until the last commits that make it compatible with 11 ...